client/src/containers/mealDetails.js

```diff
@@ -18,13 +20,30 @@ class MealDetails extends Component {
 
     return (
       <div>
-        <Link to="/meals">Back to meals</Link>
-        <h3>{this.props.meal.name}</h3>
-        <img src={this.props.meal.images} width="300" height="200" />
-        <h5> Description </h5>
-        <div>{this.props.meal.description}</div>
-        <h5>Pick Up Info</h5>
-        <div>{this.props.meal.pickupInfo}</div>
+      <Card>
+        <CardHeader
+          title="JORV"
+          subtitle="Chef"
+          avatar="http://www.ssrfanatic.com/forum/attachments/f6/68324d1235842541-funniest-avatar-troll-baby.jpg"
+        />
+        <CardMedia
+          overlay={<CardTitle title={this.props.meal.name} subtitle="JORV" />}
+        >
+          <img src={this.props.meal.images} alt="meal" width="500" height="500"/>
+        </CardMedia>
+        <CardTitle title="Meal Details" subtitle="Description"/>
+        <CardText>
+          {this.props.meal.description}
+        </CardText>
+        <CardTitle subtitle="Pickup Information"/>
+        <CardText>
+          {this.props.meal.pickupInfo}
+        </CardText>
+        <CardActions>
+          <FlatButton label="Purchase" />
+          <FlatButton label={<Link to={"/meals"} style={{color: "black", textDecoration: "none"}}>Back To Meals</Link>} />
```

Comment:
The quotes are in a JavaScript block, so should be replaced with single quotes.

Also, is there a way to rewrite this so we're not passing a component as a label?